google-cloud-contrib/google-cloud-nio-examples/README.md

```diff
@@ -21,6 +21,7 @@ To run this example:
 
 4.    Run the sample with:
 
+[//]: # ({x-version-update-start:google-cloud-nio:current})
```

Comment:
Should this be `google-cloud-nio` or `google-cloud-nio-examples`?

Reply:
Hmm this one is tricky - there is a single line which contains both versions. I'm not sure how to address this intra-line difference, besides just require that nio & nio-examples match versions.